container-setup/utils/bin/pause-hive-sync

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# OCM_CONTAINER_DOC: Adds the hive syncset-pause annotation to a cluster deployment
+
+# Annotation to pause sync
+ANNOTATION="hive.openshift.io/syncset-pause=\"true\""
+
+# _help provides standard help message for the script
+_help(){
+        echo -e "Usage: $(basename ${0}) -c CLUSTER_NAME\n"
```

Comment:
why add the `-e` if you are doing a newline anyway? is it because you are escaping any characters entered by the user? if it's not these reasons am I missing something?

Reply:
I think this was a copy/paste omission, when I moved it from another part of the script.  Thanks!